R/workhorse.R

```diff
@@ -27,57 +28,79 @@ build_request <- function(endpoint = NULL,
 ## right now not setting endpoint, if you want to include parameters, must be in a named list
 ## may introduce the :parameter notation in endpoint later :)
 
-set_query <- function(x){
-  if (length(x$params) == 0L) return(x)
+set_query <- function(x) {
+  if (length(x$params) == 0L)
+    return(x)
   if (x$method != "GET") {
     if (grepl("\\?", x$endpoint)) {
       x$query <- sub(".*\\?", "", x$endpoint)
       return(x)
     }
     return(x)
   }
-  if (!all(has_names(x$params))){
+  if (!all(has_names(x$params))) {
     spf("All parameters must be named.")
   }
   x$query <- x$params
   x$params <- NULL
   x
 }
 
-set_body <- function(x){
-  if (length(x$params) == 0L) return(x)
+set_body <- function(x) {
+  if (length(x$params) == 0L)
+    return(x)
```

Comment:
😱 I used the "Reformat Code" in RStudio and it changed those (& I didn't notice) do you generally just use the "Reindent Lines"?